### ISSUE TITLE
Documentation Issues - August 2015

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5087]
- [x] IDD notes for SwimmingPool:Indoor, Surface Name, say
  \note To be matched with a construction in this input file
  should be
  \note Name of the surface where the pool is located.  Must be a floor surface.

**Comment by mjwitte** [comment]
- [x] I/O Ref does not have a description of output:variable Availability Manager Optimum Start Control Status @hongtz68 please advise what the description should be.  Is it the same as this one?

**Availability Manager Night Ventilation Control Status []**
A value of 0 indicates the manager is signaling NoAction. A value of 2 means the manager is signaling CycleOn.
_MJW 07 Dec 2015_
Checked the source code in SystemAvailabilityManager::CalcOptStartSysAvailMgr and this manager uses the same status flags as the others:  NoAction=0, CycleOn=2.

**Comment by mjwitte** [comment]
- [x] I/O ref and IDD say "Multimode - activate enhanced dehumidification mode as needed and meet sensible load. This option is used to model DX equipment with a controllable heat exchanger assisting the DX cooling coil for improved dehumidification. It is valid only with cooling coil type = CoilSystem:Cooling:DX:HeatExchangerAssisted."

Shouldn't this include coil type Coil:Cooling:DX:TwoStageWithHumidityControlMode? 

Per @rraustad I see the control in HVACDXSystem and UnitarySystem, but not Furnaces (i.e., Furnaces doesn't even call SimDXCoilMultiMode). This surprises me since it's only a matter of setting DehumMode = 1 and then controlling to the sensible load. I would note in the issue to test UnitarySystem before changing the docs (I think I recall testing this but I can't be sure).

```
CoilSystem:Cooling:DX

   A8,  \field Dehumidification Control Type
        \type choice
        \key None
        \key Multimode
        \key CoolReheat
        \default None
        \note None = meet sensible load only
        \note Multimode = activate enhanced dehumidification mode
        \note as needed and meet sensible load. If no sensible load
        \note exists, and Run on Latent Load = Yes, and a latent
        \note load exists, the unit will operate to meet the latent load.
        \note Valid only with Coil:Cooling:DX:TwoStageWithHumidityControlMode
        \note or CoilSystem:Cooling:DX:HeatExchangerAssisted.
        \note CoolReheat = cool beyond the dry-bulb setpoint.
        \note as required to meet the humidity setpoint.
        \note Valid for all coil types. 
```

**Comment by rraustad** [comment]
Shouldn't this include coil type Coil:Cooling:DX:TwoStageWithHumidityControlMode?

I just checked and yes it should.

**Comment by mjwitte** [comment]
- [x] Helpdesk ticket 10704 Input Output Reference, per user:
  The output variable description of 'Zone Thermal Comfort ASHRAE 55 Adaptive Model Temperature [C]' in the input/output reference reads:  
  _This field reports the ideal indoor operative temperature, or comfort temperature, as determined by the ASHRAE-55 adaptive comfort model. The 80% acceptability limits for indoor operative temperature are defined as no greater than 2.5 degrees C from the adaptive comfort temperature. The 90% acceptability limits are defined as no greater than 3.5 degrees C from the adaptive comfort temperature._

I believe the 2.5 degree range applies to 90% acceptability limits and the 3.5 degree range applies to 80% acceptability limits. 
@hongtz68 Please confirm.

_MJW 07 Dec 2015_
Based on the code, these should say "less than x degrees C fro the adaptive comfort temperature:

```
                    if ( OpTemp < tComf + 2.5 && OpTemp > tComf - 2.5 ) {
                        // 80% and 90% limits okay
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5590 = 1;
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5580 = 1;
                    } else if ( OpTemp < tComf + 3.5 && OpTemp > tComf - 3.5 ) {
                        // 80% only
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5590 = 0;
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5580 = 1;
                        People( PeopleNum ).TimeNotMetASH5590 += SysTimeElapsed;
```

**Comment by mjwitte** [comment]
@hongtz68 For the ASHRAE 55 adaptive comfort model, the (corrected) output definitions say
"This field reports the ideal indoor operative temperature, or comfort temperature, as determined by the ASHRAE-55 adaptive comfort model. The 80% acceptability limits for indoor operative temperature are defined as no greater than 3.5 degrees C from the adaptive comfort temperature. The 90% acceptability limits are defined as no greater than 2.5 degrees C from the adaptive comfort temperature. A value of -1 means not applicable (when running average outdoor temp is outside the range of 10.0 to 33.5C).

But the code appears to be checking for "less than" rather than "less than or equal to"

```
                    if ( OpTemp < tComf + 2.5 && OpTemp > tComf - 2.5 ) {
                        // 80% and 90% limits okay
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5590 = 1;
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5580 = 1;
                    } else if ( OpTemp < tComf + 3.5 && OpTemp > tComf - 3.5 ) {
                        // 80% only
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5590 = 0;
                        ThermalComfortData( PeopleNum ).ThermalComfortAdaptiveASH5580 = 1;
                        People( PeopleNum ).TimeNotMetASH5590 += SysTimeElapsed;
```

Would you please confirm what the standard says?  

_MJW 15 Dec 2015_ This question is now #5372

**Comment by mjwitte** [comment]
Closed via #5352
